config/systemd-monitor.json

```diff
@@ -0,0 +1,28 @@
+{
+	"plugin": "journald",
+	"pluginConfig": {
+		"source": "systemd"
+	},
+	"logPath": "/var/log/journal",
+	"lookback": "",
+	"bufferSize": 10,
+	"source": "systemd-monitor",
+	"conditions": [],
+	"rules": [
+		{
+			"type": "temporary",
+			"reason": "KubeletRestart",
```

Comment:
@Random-Liu, should it be `KubeletStart` instead of `KubeletRestart`? WDYT?

Reply:
Does Kubelet restart and start logs different messages? According to the log, it seems not. So,  i think `KubeletStart` should be reasonable. But users should diagnose which `KubeletStart` event represents a normal Kubelet start and which `KubeletStart` event represents a abnormal restart.